ui/src/Components/SearchCandidates.jsx

```diff
@@ -61,37 +66,57 @@ export default class SearchCandidates extends React.Component {
+						this.populate(items, obj, i)
+					}
+				}
+				else if (this.state.searchName != null && this.state.searchEmail != null && this.state.searchVerified == true) {
+					if (obj[i].first_name.concat(' ', obj[i].last_name).toLowerCase().includes(this.state.searchName) && obj[i].email.toLowerCase().includes(this.state.searchEmail) && obj[i].app_status.toLowerCase().includes('completed validation')) {
+						this.populate(items, obj, i)
 					}
 				}
-				else if (this.state.searchName != null && this.state.searchNumber != null) {
-					if ((obj[i].first_name.toLowerCase().includes(this.state.searchName) || obj[i].last_name.toLowerCase().includes(this.state.searchName)) && obj[i].contact_num.includes(this.state.searchNumber)) {
+				else if (this.state.searchName != null && this.state.searchEmail != null && this.state.searchPendingVerified == true) {
+					if (obj[i].first_name.concat(' ', obj[i].last_name).toLowerCase().includes(this.state.searchName) && obj[i].email.toLowerCase().includes(this.state.searchEmail) && obj[i].app_status.toLowerCase().includes('pending validation')) {
 						this.populate(items, obj, i)
 					}
 				}
 				else if (this.state.searchName != null && this.state.searchEmail != null) {
-					if ((obj[i].first_name.toLowerCase().includes(this.state.searchName) || obj[i].last_name.toLowerCase().includes(this.state.searchName)) && obj[i].email.toLowerCase().includes(this.state.searchEmail)) {
+					if (obj[i].first_name.concat(' ', obj[i].last_name).toLowerCase().includes(this.state.searchName) && obj[i].email.toLowerCase().includes(this.state.searchEmail)) {
+						this.populate(items, obj, i)
+					}
+				}
+				else if (this.state.searchName != null && this.state.searchVerified == true) {
+					if (obj[i].first_name.toLowerCase().includes(this.state.searchName) && obj[i].app_status.toLowerCase().includes('completed validation')) {
+						this.populate(items, obj, i)
+					}
+				}
+				else if (this.state.searchName != null && this.state.searchPendingVerified == true) {
+					if (obj[i].first_name.toLowerCase().includes(this.state.searchName) && obj[i].app_status.toLowerCase().includes('pending validation')) {
 						this.populate(items, obj, i)
 					}
 				}
-				else if (this.state.searchNumber != null && this.state.searchEmail != null) {
-					if (obj[i].contact_num.includes(this.state.searchNumber) && obj[i].email.toLowerCase().includes(this.state.searchEmail)) {
+				else if (this.state.email != null && this.state.searchVerified == true) {
+					if (obj[i].email.toLowerCase().includes(this.state.searchEmail) && obj[i].app_status.toLowerCase().includes('completed validation')) {
 						this.populate(items, obj, i)
 					}
 				}
-				else if (this.state.searchNumber != null) {
-					if (obj[i].contact_num.includes(this.state.searchNumber)) {
+				else if (this.state.email != null && this.state.searchPendingVerified == true) {
```

Comment:
Only shows the first clicked status
![image](https://user-images.githubusercontent.com/37027135/101666480-1cf40880-3a03-11eb-8af6-ba430314fc52.png)
![image](https://user-images.githubusercontent.com/37027135/101666490-22e9e980-3a03-11eb-9210-73525abbe868.png)
Notice how when both are clicked nothing changed but when one is un-clicked the latter shows up
![image](https://user-images.githubusercontent.com/37027135/101666561-3a28d700-3a03-11eb-8dc0-ed70c59b0db7.png)